config/manager/manager_image_patch.yaml

```diff
@@ -8,6 +8,5 @@ spec:
     spec:
       containers:
       # Change the value of image field below to your controller image URL
-      - image: cluster-api-do-controller
+      - image: gcr.io/k8s-staging-cluster-api-do/cluster-api-do-controller
```

Comment:
Why we use staging instead of prod image here? For what's this image used exactly?

Reply:
The image here is a just default image so when some users/developers want to install this provider without building the image first they can use this manifest using a staging image (which is the current branch master state).

The other provider only uses production image in release manifest so i think we need to have a similar implementation.

Reply:
Sounds good!